processing/src/main/java/io/druid/query/Queries.java

```diff
@@ -20,42 +20,64 @@
 package io.druid.query;
 
 import com.google.common.base.Preconditions;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
 import com.google.common.collect.Sets;
 import io.druid.query.aggregation.AggregatorFactory;
 import io.druid.query.aggregation.PostAggregator;
 
 import java.util.List;
+import java.util.Map;
 import java.util.Set;
 
 /**
  */
 public class Queries
 {
-  public static void verifyAggregations(
+  public static List<PostAggregator> decorate(List<PostAggregator> postAggs,
```

Comment:
minor: name this `decoratePostAggregators`? Or move to `PostAggregators.decorate`? The name "Queries.decorate" sounds a little too generic for what this does.